content/docs/howto/php.md

```diff
@@ -61,113 +64,352 @@ app, include the values below in your `composer.json` file:
+You can configure which web server to use by setting the `BP_PHP_SERVER` environment variable at build-time.
+The setting options are `php-server`, `httpd`, `nginx`, with the default value of `php-server` if unset.
+{{< code/copyable >}}
+  pack build my-app --buildpack paketo-buildpacks/php \
+  --env BP_PHP_SERVER="php-server"
+{{< /code/copyable >}}
+
+#### Deprecated: buildpack.yml configuration
+Note that specifying any configurations through `buildpack.yml` configuration
+will be deprecated in PHP language-family buildpack v1.0.0.
+To migrate from using `buildpack.yml` please set the environment variable
+equivalents instead.
+
+## Set server configuration
+If you're using `httpd` or `nginx`, a suitable `httpd.conf` or `nginx.conf`
+will be generated for you by the buildpack and made available in a layer.
+Check out the [PHP Reference documentation]({{< ref
+"docs/reference/php-reference" >}}) for an enumeration of the defaults set for
+`httpd` and `nginx`.
+
+Additional configuration can also be provided via environment variables.
+
+### Provide your own web server configuration file
+If either `httpd` or `nginx` are the selected web server, users can provide
+their own configurations in the form of a server-specific configuration file.
+
+#### Provide `httpd` specific configuration
+In the `httpd` case, user-included configuration can be placed be found in the
+application source directory under `<app-directory>/.httpd.conf.d/*.conf`.
+
+#### Provide `nginx` specific configuration
+In the `nginx` case, user-included configuration can be placed in the
+application source directory under `<app-directory>/.nginx.conf.d/`.
+Server-specific configuration should be inside a file named  `*-server.conf`,
+and HTTP configuration should be inside a file with the naming structure
+`*-http.conf`.
+
+### Configure FPM settings
+The PHP buildpack includes support for the [PHP FastCGI Process
+Manager][external/fpm] (FPM) when used in conjunction with a web server. In
+this case, the [PHP FPM buildpack][bp/fpm] will generate FPM configuration for
+you to work with the web server of choice. On top of setting up FPM
+configuration, the buildpack will consider configuration from:
+
+- PHP  (in the PHP Dist buildpack)
+- other buildpacks (such as PHP Nginx)
+- the user (see `Supply user-provided FPM configuration` section below)
+
+Check out the [PHP Reference documentation]({{< ref
+"docs/reference/php-reference" >}}) for an enumeration of the defaults set for
+FPM.
+
+#### Supply user-provided FPM configuration
+Users can provide FPM configuration by providing a configuration file in the
+application source directory under `.php.fpm.d/*.conf`.
+
+User-provided configuration will be considered the highest precedence source of
+configuration, and should be provided in an `.ini` compliant format in order to
+be considered by PHP FPM.
+
+### Configure the web directory
+The PHP web directory for the `httpd` and `nginx` server cases can be
```

Comment:
What is the PHP web directory? Googling didn't reveal anything compelling.

Reply:
added some context. I had it wrong here.